### ISSUE TITLE
update to WP 4.1

### ISSUE THREAD

**Issue by ericgj** [GoogleCloudPlatform/appengine-php-wordpress-starter-project#40]
I was considering using this as a template but noticed it hasn't been updated in 7 months and is still at WP 3.9.1  :cry:  And the WP sources are included as zip files  :sob:   Is there a newer repo somewhere?

You might consider an approach like https://github.com/xyu/heroku-wp which uses PHP Composer to handle WP and plugin dependencies. Of course unlike Heroku, GAE doesn't allow custom install scripts, but at least you could dynamically install and lock the deps locally before deploying.

**Comment by dvhirst** [comment]
Eric,

I worked with this template from October 2014 thru January 2015 without
success.  I finally gave up and moved to the Amazon Web Service platform;
was able to implement WP without any issues, am close to having CiviCRM set
up on WP (using WP as a CMS), and able to access it with HTTPS (TLS
protected).  So far, no insurmountable problems.  Good luck.

DVHirst

**Comment by phamuelson** [comment]
Hi @ericgj From the project page, I clicked on the "wordpress" folder and it looks like its contents get updated directly from WordPress page, so it will load WP 4.2.
GAE Project:  https://github.com/GoogleCloudPlatform/appengine-php-wordpress-starter-project
**They should fix the CL summary as it makes it look like the last update was 7 mos ago.

Sidenote: I was successful to get this working in Jan 2015, but one thing I remember having to do was ignoring the section about running move_files_after_editing.sh ; instead, I moved the files manually because it wasn't moving it correctly (perhaps due to my local env).
